components/icons/catalogue.md

```diff
@@ -38,9 +38,10 @@ Size: `18px` x `18`px
-| ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/accelerator.svg?sanitize=true) | accelerator | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/eportfolio.svg?sanitize=true) | eportfolio | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/plus-large-thick.svg?sanitize=true) | plus-large-thick |
-| ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/access-special.svg?sanitize=true) | access-special | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/event-log.svg?sanitize=true) | event-log | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/plus-large.svg?sanitize=true) | plus-large |
-| ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/add-file.svg?sanitize=true) | add-file | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/exemption-add.svg?sanitize=true) | exemption-add | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/preview.svg?sanitize=true) | preview |
+| ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/accelerator.svg?sanitize=true) | accelerator | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/enrollment.svg?sanitize=true) | enrollment | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/plus-default.svg?sanitize=true) | plus-default |
+| ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/access-special.svg?sanitize=true) | access-special | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/eportfolio.svg?sanitize=true) | eportfolio | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/plus-large-thick.svg?sanitize=true) | plus-large-thick |
+| ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/add-file.svg?sanitize=true) | add-file | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/event-log.svg?sanitize=true) | event-log | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/plus-large.svg?sanitize=true) | plus-large |
+| ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/add-filled.svg?sanitize=true) | add-filled | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/exemption-add.svg?sanitize=true) | exemption-add | | ![](https://raw.githubusercontent.com/BrightspaceUI/core/main/components/icons/images/tier1/preview.svg?sanitize=true) | preview |
```

Comment:
This adds the `add-filled` tier1 icon and modifies the `add` icon.
There are also `tier2` versions but they needed tweaks so I'll be updating that later.

Reply:
Update on this: we don't yet want to replace the existing `add` icon so I'm going to make this a new file.